controllers/ovncontroller_controller.go

```diff
@@ -576,6 +571,13 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 
 	if instance.Status.NumberReady == instance.Status.DesiredNumberScheduled {
 		instance.Status.Conditions.MarkTrue(condition.DeploymentReadyCondition, condition.DeploymentReadyMessage)
+	} else {
+		instance.Status.Conditions.Set(condition.FalseCondition(
+			condition.DeploymentReadyCondition,
+			condition.RequestedReason,
+			condition.SeverityInfo,
+			condition.DeploymentReadyRunningMessage))
+		return ctrl.Result{RequeueAfter: time.Second * 10}, nil
```

Comment:
I don't think we need to actively poll this. The controller owns the dameonset it creates  
https://github.com/openstack-k8s-operators/ovn-operator/blob/cfd21c1957b75c1e1d95c1de35203a2f3e47c0a0/controllers/ovncontroller_controller.go#L239 so it will be reconciled automatically when the status of the daemonset changes.

Reply:
agree, I did not intend to leave this bit in from an earlier test